sdk/dotnet/Network/Inputs/FirewallApplicationRuleCollectionRuleProtocolGetArgs.cs

```diff
@@ -15,8 +15,8 @@ public sealed class FirewallApplicationRuleCollectionRuleProtocolGetArgs : Pulum
         /// <summary>
         /// Specify a port for the connection.
         /// </summary>
-        [Input("port")]
-        public Input<int>? Port { get; set; }
+        [Input("port", required: true)]
+        public Input<int> Port { get; set; } = null!;
```

Comment:
Breaking change, but it apparently became required upstream, so not much we can do about it.